docs/prompt_injection/jailbreaking.md

```diff
@@ -0,0 +1,49 @@
+---
+sidebar_position: 4
+---
+
+# Jailbreaking
+
+In the case of Large Language Models (LLMs), jailbreaking refers to generating prompts to bypass the safety features placed by its creators to generate unauthorized responses(@perez2022jailbreak)(@brundage_2022)(@wang2022jailbreak).
+
+## Methodologies of Jailbreaking
+
+OpenAI, among other companies or organizations that create LLMs, include content moderation to ensure their models do not produce controversial (violent, sexual, illegal, etc.) responses(@markov_2022)(@openai_api). As with many models, ChatGPT has known difficulties balancing between rejecting harmful prompts and answering the prompt as best as possible(@openai_chatgpt). Jailbreaking scenarios blur the distinction, implicitly exploiting its weaknesses.

```

Comment:
can you clarify this section? 
> answering the prompt as best as possible

What does this mean?

> Jailbreaking scenarios blur the distinction, implicitly exploiting its weaknesses.

I don't quite understand this